client-v2/README.md

```diff
@@ -28,7 +28,7 @@ View [setup.sh](scripts/setup.sh).
 
 ## Developing
 
-To start developing just run:
+To start developing, first get [Janus credentials for `cmsFronts`](https://janus.gutools.co.uk/credentials?permissionId=cmsFronts-dev&tzOffset=0) then run:
```

Comment:
You also need CAPI gateway invocation credentials and dotcom credentials if you want to be able to press fronts from your local app: https://github.com/guardian/facia-tool/blob/master/README.md#credentials

Reply:
oh, oops! Fixed [here](https://github.com/guardian/facia-tool/pull/525)